README.md

```diff
@@ -14,9 +14,13 @@ Role Variables
 
 | Name              | Default Value       | Description          |
 |-------------------|---------------------|----------------------|
-| `rhel7stig_cat1_patch` | True | Correct CAT I findings |
-| `rhel7stig_cat2_patch` | False | Correct CAT II findings |
-| `rhel7stig_cat3_patch` | False | Correct CAT III findings |
+| `rhel7stig_cat1_audit` | True  | Audit for CAT I findings      |
```

Comment:
Please set the default values to `yes` or `no` and wrap them in backticks so they are formatted as code.

    | `rhel7stig_cat1_audit` | `yes`  | Audit for CAT I findings      |